### ISSUE TITLE
Property type array returns string

### ISSUE THREAD

**Issue by jovicacosmic** [zircote/swagger-php#926]
```
/**
     * @OA\Post(path="/api/categories", tags={"Categories"}, summary="Create category",
     *   @OA\RequestBody(description="Input data", required=true,
     *      @OA\MediaType(mediaType="multipart/form-data",
     *         @OA\Schema(type="object", required={ "name", "status", "position", "attributes"},
     *            @OA\Property(property="name", type="string"),
     *            @OA\Property(property="status", type="integer"),
     *            @OA\Property(property="position", type="integer"),
     *            @OA\Property(property="attributes", type="array",
     *               @OA\Items(type="integer", example="11")
     *            ),
     *            @OA\Property(property="image", type="string", format="binary"),
     *         ),
     *      ),
     *   ),
     *   @OA\Response(response=201, description="Success"),
     *   @OA\Response(response=401, description="Unauthenticated"),
     *   @OA\Response(response=400, description="Bad Request"),
     *   @OA\Response(response=404, description="Not found"),
     *   @OA\Response(response=403, description="Forbidden"),
     *   security={{"bearerAuth": {}}}
     *)
     **/
```

Am i doing something wrong? (Using Laravel 8)

Attributes fields returns string`"[11,12]"`

Please help :D

**Comment by DerManoMann** [comment]
> Attributes fields returns string"[11,12]"

Is this in your PHP code? If so then you are looking at how swagger-ui is submitting the data.
Having said that, form data is always of type string, unless I am mistaken?

**Comment by KalimeroMK** [comment]
Pls explain don't understand your answer

**Comment by DerManoMann** [comment]
Not so much an answer but a question - the example annotation you posted result for me in this OpenApi spec:
```yaml
openapi: 3.0.0
paths:
  /api/categories:
    post:
      tags:
        - Categories
      summary: 'Create category'
      requestBody:
        description: 'Input data'
        required: true
        content:
          multipart/form-data:
            schema:
              required:
                - name
                - status
                - position
                - attributes
              properties:
                name:
                  type: string
                status:
                  type: integer
                position:
                  type: integer
                attributes:
                  type: array
                  items: { type: integer, example: '11' }
                image:
                  type: string
                  format: binary
              type: object
      responses:
        '201':
          description: Success
        '401':
          description: Unauthenticated
        '400':
          description: 'Bad Request'
        '404':
          description: 'Not found'
        '403':
          description: Forbidden
      security:
        -
          bearerAuth: []
```
If I load that in swagger-ui I can add one or more attribute input fields to the request and those are prepopulated with `11`

I do not think that is quite what is intended but then I am not sure the `example` property is actually valid inside the `items` object in the generated spec.

So, coming back to your original question - where do you see the `"[11,12]"`?

**Comment by TarasMoroz** [comment]
I've met with the same problem. Does anyone know a solution? 

`     *     @OA\RequestBody(
     *         @OA\MediaType(
     *             mediaType="application/x-www-form-urlencoded",
     *             @OA\Schema(
     *                 @OA\Property(
     *                     property="lang_id[]",
     *                     description="Lang id for text translation (watch /languages endpoint for retrieving IDs)",
     *                     type="array",
     *                     @OA\Items(type="integer", example="1")
     *                 ),
     *                 @OA\Property(
     *                     property="lang_text[]",
     *                     description="Translation text for related language id in lang_id[]",
     *                     type="array",
     *                     @OA\Items(type="string", example="Text in Swedish")
     *                 )
     *             )
     *         )
     *     ),`
     
     At swagger UI I have added lang_id array items as 1 and 2. The same with lang_text array: string1, string2. And what i have in php, when I made var_dump() is
     
     `array(2) {
        ["lang_id"]=>
        array(1) {
          [0]=>
          string(3) "1,2"
        }
        ["lang_text"]=>
        array(1) {
          [0]=>
          string(3) "string1,string2"
        }
      }`

  One more thing, my CURL request looks like that: 

`curl -X 'PUT' \
  'http://localhost:8484/syntax/questions/10' \
  -H 'accept: */*' \
  -H 'Content-Type: application/x-www-form-urlencoded' \
  -d 'lang_id%5B%5D=1,2&lang_text%5B%5D=d,d'`
